must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -259,6 +259,8 @@ for ns in $namespaces; do
                 { 
                     printf "Collecting image info for: %s/%s\n" "${bp}" "${image}";
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd info $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log;
+                    printf "Collecting image status for: %s/%s\n" "${bp}" "${image}";
+                    timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd status $image --pool $bp" 2>> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-image-"${image}"-debug.log;
```

Comment:
Should the stderr "gather" file be named differently from the one used for `rbd info`?  Not sure what the convention is, but different files are used for `rbd snap ls --all` and `rbd snap ls --all --format=json` below.

Reply:
I wouldn't know, but the filename has been updated now.